src/Persistence_matrix/include/gudhi/Persistence_matrix/allocators/entry_constructors.h

```diff
@@ -0,0 +1,139 @@
+/*    This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
+ *    See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full license details.
+ *    Author(s):       Hannah Schreiber
+ *
+ *    Copyright (C) 2024 Inria
+ *
+ *    Modification(s):
+ *      - YYYY/MM Author: Description of the modification
+ */
+
+/**
+ * @file entry_constructors.h
+ * @author Hannah Schreiber
+ * @brief Contains different versions of @ref Gudhi::persistence_matrix::Entry factories.
+ */
+
+#ifndef PM_COLUMN_CELL_CONSTRUCTORS_H
+#define PM_COLUMN_CELL_CONSTRUCTORS_H
```

Comment:
Maybe rename also the directives:
```suggestion
#ifndef PM_COLUMN_ENTRY_CONSTRUCTORS_H
#define PM_COLUMN_ENTRY_CONSTRUCTORS_H
```

Reply:
Right, I forgot them! Should be done now.